src/view_controls/view.py

```diff
@@ -524,16 +525,14 @@ def shown(self):
         """
             We should show if the following is true:
                 1. We are showable
-                2. We are guppy
-                3. We are health pickup AND we want to see health pickups
-                4. We are rerolled AND we want to see rerolls
-                5. We are a spacebar AND we want to see spacebars
+                2. We are health pickup AND we want to see health pickups
+                3. We are rerolled AND we want to see rerolls
+                4. We are a spacebar AND we want to see spacebars
+                Note: Removed Guppy check as it doesn't seem necessary here, as a result I didn't add any of the transformations
```

Comment:
About the note : I believe the aim of the guppy check was to bypass the "hide spacebar items" option, and always display guppy touch for head and paw.